### DISCUSSION TITLE
水星記MV中的機器人

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#1127] [閒聊八卦]
不知道有多少人看過水星記的MV
因為角色切換很快所以我聽了好幾遍但還是沒有全懂
突然想到 就認真的去查了
—
看完文章後我不斷的思考
用虛擬實境讓人們看已經不存在的事物
對失憶的人們來說究竟是福音
還是一種傷痛呢
那是一段很傷心很傷心的故事
在幻境中什麼都有 關於不在的人們 曾經的往事
在幻境中有多快樂 回歸現實的時候就有多痛苦
—
若今天失憶的人是你，你會選擇承擔這份痛苦而在所不惜的去回憶嗎
又或是寧願不回想起往事，繼續著原先的生活呢
歡迎跟我分享你的選擇！
https://vocus.cc/article/5eeb6b46fd897800011adb2f

**Comment by Sylvie0601** [comment]
我會想要承擔ㄟ，因為我覺得就算是再痛苦的記憶也是我的記憶，失去記憶本身就是一件很悲傷的事情，我會想盡辦法回憶，我很怕我失去的是我很愛的人，就算那些回憶有很多都是痛苦的，但我還是想要保留，因為那些記憶裡面說不定有一些很小的美好，我不想要丟失的那一塊。

題外話 : 水星記真的很好聽
